docker/vm/nodejs/nginx_vm.conf

```diff
@@ -0,0 +1,84 @@
+upstream django_app {
+  server django:8000 fail_timeout=0;
+}
+
+# Django server
+server {
+  # Listen on port 80 & then redirect to SSL port
+  listen 80;
+  server_name BACKEND;
+
+  # redirection to SSL 
+  location / {
+    return 301 https://$host$request_uri;
+  }
+
+  # Certbot challenge files
+  location /.well-known/acme-challenge/ {
+    root /var/www/certbot;
+  }
+}
+
+server {
+  listen 443 ssl;
+  server_name BACKEND;
+  client_max_body_size 400M;
+
+  # SSL certificates generated by init-letsencrypt
+  # Don't change the FRONTEND. Certbot places certificates in the first domain passed in init-letsencrypt. 
+  # See https://certbot.eff.org/docs/using.html#where-are-my-certificates to know more
+  ssl_certificate /etc/letsencrypt/live/FRONTEND/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/FRONTEND/privkey.pem;
+  include /etc/letsencrypt/options-ssl-nginx.conf;
+  ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
+
+  # Access Logs
+  access_log /var/log/nginx/access.log;
+  error_log /var/log/nginx/error.log;
+  #Redirect to django server on 8000
+  location / {
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $http_host;
+    proxy_pass http://django_app/;
+  }
+}
+
+# Nodejs
+server {
+  listen 80;
+  server_name FRONTEND;
+
+  location / {
+    return 301 https://$host$request_uri;
+  }
+
+  location /.well-known/acme-challenge/ {
+    root /var/www/certbot;
+  }
+}
+
+server {
+  listen 443 ssl;
+  sendfile on;
+  default_type application/octet-stream;
+  
+  server_name FRONTEND;
```

Comment:
instead of using `FRONTEND` or `BACKEND` use `FRONTEND_DOMAIN` `BACKEND_DOMAIN`

Reply:
Done